test/lint/lint-includes.sh

```diff
@@ -54,8 +54,6 @@ EXPECTED_BOOST_INCLUDES=(
     boost/algorithm/string/replace.hpp
     boost/algorithm/string/split.hpp
     boost/date_time/posix_time/posix_time.hpp
-    boost/filesystem.hpp
-    boost/filesystem/fstream.hpp
```

Comment:
Removing of `boost/filesystem/fstream.hpp` should be done in the _"refactor: replace boost::filesystem with std::filesystem"_ commit (5fb92bab5f8c6c07bb668d7a7f2203f389a4956d) to let it pass `test/lint/lint-includes.sh`.